### ISSUE TITLE
Searchable plaintext export mishandling some whitespace

### ISSUE THREAD

**Issue by mulcare** [benwbrum/fromthepage#1712]
The "searchable plaintext" that a IIIF export points to is mishandling some whitespace. Two carriage returns between text, specifically, behaves oddly. Compare the [first few lines of this page](https://fromthepage.com/harvardlibrary/colonial-north-america-harvard-university-archives/bentley-william-1759-1819-papers-of-william-bentley-1783-1815-an-inventory-letter-from-james-winthrop-to-william-bentley-1812-may-4-hug-1203-5-box-1-harvard-university-archives/transcribe/1050407) to the [searchable plaintext export](https://fromthepage.com/iiif/31971/export/plaintext/searchable).

Returned:
`Cambridge 4 May 1812{Reverend} Sir,Your very learned...`

Expected:
`Cambridge 4 May 1812 {Reverend} Sir, Your very learned...`

**Comment by saracarl** [comment]
A similar report from another customer:

_One of our staff just reported this. I confirmed that it is different than what we see in the Browse display.
On the search results display, there are no spaces between words separated by line breaks. In Browse, the spaces are shown.

Search Example: https://fromthepage.com/display/search?utf8=%E2%9C%93&collection_id=631&search_string=telescope&button=
Browse Example: https://fromthepage.com/whslamc/increase-lapham/box-10-folder-2-correspondence-june-december-1865?page=19
Page Display: https://fromthepage.com/whslamc/increase-lapham/box-10-folder-2-correspondence-june-december-1865/display/889997

I have not verified what happens with the text that is re-imported to Contentdm._

**Comment by benwbrum** [comment]
This is a problem in `SearchTranslator.search_text_from_xml` which needs similar logic to this (from AbstractXmlController):
```ruby

      if preserve_lb
        e.replace_with(REXML::Element.new('br'))
      else
        if params[:action] == "read_work" || params[:action] == 'needs_review_pages' || params[:action] == 'page_plaintext_searchable' || params[:action] == 'work_plaintext_searchable'
          if e.attributes['break'] == "no"
            lb.add_text('')
          else
            lb.add_text(' ')
            lb.add_attribute('class', 'line-break')
          end
        else
          if e.attributes['break'] == "no"
            lb.add_text('-')
          end
        end
      end
```

**Comment by benwbrum** [comment]
Note above the addition of the `' '` to the element if there is no `break="no"` attribute in the `lb` element.

**Comment by benwbrum** [comment]
I believe that this was closed by #1812 or #1943.  @mulcare would you mind verifying that the behavior now matches your expectations?

**Comment by benwbrum** [comment]
We continue to experience problems with line breaks; these should be replaced with spaces in the searchable export, but are not.

See https://fromthepage.com/iiif/31971/export/plaintext/searchable and the following text, which should have spaces between `safe` and `during`, `no` and `need`, and possibly `1812` and `{Reverend}`.

```
Cambridge 4 May 1812{Reverend} Sir,Your very learned letter of 27 April, arrived safeduring my absence, & on my return I received it. There will be noneed of adding that it gave much satisfaction as all your c
```
